reposilite-site/docs/reverse-proxy-ssl.md

```diff
@@ -0,0 +1,87 @@
+---
+id: reverse-proxy-ssl
+title: Reverse proxy ssl
+sidebar_label: Reverse proxy ssl
+---
+
+Lots of people like to use a reverse proxy like nginx with reposilite. This is a page on
+how to setup Nginx with SSL.
+
+Source: <https://gist.github.com/Saladoc/8c17bda7b131c4fade9b2189ea5050d8>
```

Comment:
Can we remove this? The gist is not meant to be kept up to date. In fact, I'll most likely delete it once this pr is merged.